jax/_src/numpy/lax_numpy.py

```diff
@@ -1986,15 +1986,18 @@ def in1d(ar1, ar2, assume_unique=False, invert=False):
   else:
     return (ar1[:, None] == ar2[None, :]).any(-1)
 
-_SETDIFF1D_DOC = """\
-Because the size of the output of ``setdiff1d`` is data-dependent, the function is not
-typically compatible with JIT. The JAX version adds the optional `size` argument which
-specifies the size of the output array: it must be specified statically for ``jnp.setdiff1d``
-to be compiled with non-static operands. If specified, the first `size` unique elements will
-be returned; if there are fewer unique elements than `size` indicates, the return value will
-be padded with the `fill_value`, which defaults to zero."""
-
-@_wraps(np.setdiff1d, lax_description=_SETDIFF1D_DOC)
+@_wraps(np.setdiff1d,
+  lax_description=_dedent("""
+    Because the size of the output of ``setdiff1d`` is data-dependent, the function is not
+    typically compatible with JIT. The JAX version adds the optional ``size`` argument which
+    must be specified statically for ``jnp.setdiff1d`` to be used within JAX transforms."""),
```

Comment:
How about:
```suggestion
    must be specified statically for ``jnp.setdiff1d`` to be used within some of JAX's transformations."""),
```
and the same for similar instances below? Transformations like JVP take no issue.